java/java-guestbook/kubernetes-manifests/guestbook-frontend.deployment.yaml

```diff
@@ -28,3 +28,5 @@ spec:
           value: "8080"
         - name: GUESTBOOK_API_ADDR
           value: java-guestbook-backend:8080
+        - name: JAVA_TOOL_OPTIONS
```

Comment:
Is this needed for the non Skaffold debug workflow? If so, did you confirm that Skaffold debug is fine if this is already present (I’m pretty sure it is).

Reply:
Yes, I tried skaffold debug with this setting and it works as well.